apps/web/next.config.js

```diff
@@ -464,7 +469,7 @@ const nextConfig = {
         },
         {
           source: "/icons/sprite.svg",
-          headers: [CORP_CROSS_ORIGIN_HEADER],
+          headers: [CORP_CROSS_ORIGIN_HEADER, ACCESS_CONTROL_ALLOW_ORIGIN_HEADER],
```

Comment:
This fixes the issue of icons not appearing locally for org bookings.

Production endpoint already has this header, I don't know why(maybe cloudflare) 
![image](https://github.com/user-attachments/assets/f0fb6e1b-7459-49bf-9150-9a91d3417227)